src/options.ts

```diff
@@ -261,6 +263,10 @@ export async function loadOptions (configOverrides: NitroConfig = {}): Promise<N
   // Resolve plugin paths
   options.plugins = options.plugins.map(p => resolvePath(p, options))
 
+  if (isWindows) {
```

Comment:
It might be worth confirming whether we need to do this _when building_ as well as in development.